spring-cloud-netflix-core/src/main/resources/META-INF/spring.factories

```diff
@@ -7,10 +7,13 @@ org.springframework.cloud.netflix.feign.encoding.FeignContentGzipEncodingAutoCon
 org.springframework.cloud.netflix.hystrix.HystrixAutoConfiguration,\
 org.springframework.cloud.netflix.ribbon.RibbonAutoConfiguration,\
 org.springframework.cloud.netflix.rx.RxJavaAutoConfiguration,\
-org.springframework.cloud.netflix.metrics.servo.ServoMetricsAutoConfiguration
-
+org.springframework.cloud.netflix.metrics.servo.ServoMetricsAutoConfiguration,\
+org.springframework.cloud.netflix.zuul.filters.pre.ratelimit.RateLimitConfiguration
+org.springframework.cloud.bootstrap.BootstrapConfiguration=\
```

Comment:
From the main branch I guess: https://github.com/spring-cloud/spring-cloud-netflix/blob/master/spring-cloud-netflix-core/src/main/resources/META-INF/spring.factories#L10